### PR TITLE
Completed "Remove unused intro files #1911" Issue

### PR DESCRIPTION
**Pull request by uroshima**
Completed "Remove unused intro files #1911" Issue

### CODE REVIEW

File: src/sass/components/_intro-story.scss [hunk 1]
```diff
@@ -125,17 +118,6 @@
 }
 
 
-.background--image1 {
-  @include breakpoints (small) {
-    background-image: url('/img/global/intro-story/slide1-flagpole-mountains-698x600.jpg');
```
Comment (DaleMcGrew) [inline src/sass/components/_intro-story.scss]: Can this image (and some of the others) be removed from /img/global/intro-story/?

**Comment by DaleMcGrew** [comment]
Thank you @uroshima, I am reviewing this now!

**Comment by uroshima** [comment]
Ok I believe now is all fixed and good to go. Thanks @DaleMcGrew

**Comment by DaleMcGrew** [comment]
👍 Thank you for your first pull request @uroshima!

**Pull request merged**